enginetest/memory_engine_test.go

```diff
@@ -558,6 +558,10 @@ func TestWindowRangeFrames(t *testing.T) {
 	enginetest.TestWindowRangeFrames(t, enginetest.NewDefaultMemoryHarness())
 }
 
+func TestNamedWindows(t *testing.T) {
```

Comment:
Don't forget to follow up and add these calls to dolt_engine_test.go in another pass

Probably worthwhile to do an audit to make sure dolt has all test methods the memory db does

Reply:
whoops yeah, will PR that before merging